### PR TITLE
Expertise edit

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/components/Sidebar/ObjectExpertise.js [hunk 1]
```diff
@@ -45,14 +54,21 @@ const ObjectExpertise = ({ username, wobject }) => {
           </React.Fragment>
         )}
 
-        {_.size(users) > 5 && (
-          <React.Fragment>
-            <h4 className="ObjectExpertise__more">
+        {_.size(users) >= 5 && (
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertise.js]: Replace literal with const

File: src/client/components/Sidebar/ObjectExpertise.js [hunk 2]
```diff
@@ -13,9 +13,18 @@ const ObjectExpertise = ({ username, wobject }) => {
   const [experts, setExperts] = useState({ user: {}, users: [] });
   const { users, user } = experts;
   const isUserInTopFive = users.find(u => u.name === username);
+  const [limit, setLimit] = useState(5);
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertise.js]: Are limit and users.length the same value? Do we need limit variable?

File: src/client/components/Sidebar/ObjectExpertise.js [hunk 3]
```diff
@@ -45,14 +54,21 @@ const ObjectExpertise = ({ username, wobject }) => {
           </React.Fragment>
         )}
 
-        {_.size(users) > 5 && (
-          <React.Fragment>
-            <h4 className="ObjectExpertise__more">
+        {_.size(users) >= 5 && (
+          <div className="ObjectExpertise__buttons-wrap">
+            <div className="ObjectExpertise__buttons-wrap-all">
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertise.js]: rename class. Use 'ObjectExpertise__show-all-btn'

File: src/client/components/Sidebar/ObjectExpertise.js [hunk 4]
```diff
@@ -45,14 +54,21 @@ const ObjectExpertise = ({ username, wobject }) => {
...
               <Link to={`/object/${wobject.author_permlink}/expertise`}>
-                <FormattedMessage id="show_more_authors" defaultMessage="Show more authors" />
+                <FormattedMessage id="show_all" defaultMessage="Show all" />
               </Link>
-            </h4>
-          </React.Fragment>
+            </div>
+            <div
+              className="ObjectExpertise__buttons-wrap-more"
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertise.js]: rename to 'ObjectExpertise__show-more-btn'

File: src/client/components/Sidebar/ObjectExpertise.less [hunk 5]
```diff
@@ -1,11 +1,27 @@
...
+  &__buttons-wrap {
+    display: flex;
+    justify-content: space-around;
+    padding-top: 14px;
 
-  & > a {
-    color: @blue-botticelli;
-    font-weight: 500;
+    &-all {
+      & > a {
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertise.less]: Select elements by class (add classname to link)

File: src/client/components/Sidebar/ObjectWeightBlock.js [hunk 6]
```diff
@@ -35,6 +36,18 @@ class ObjectWeightBlock extends React.Component {
     }
   }
 
+  showMoreExpertise = () => {
+    this.setState({ limit: this.state.limit + 5 }, () => {
+      api.getWobjectsWithUserWeight(this.props.username, 0, this.state.limit).then(data =>
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectWeightBlock.js]: change skip, not limit

File: src/client/components/Sidebar/ObjectExpertise.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/components/Sidebar/ObjectExpertise.js]: ++

File: src/client/components/Sidebar/ObjectExpertise.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/components/Sidebar/ObjectExpertise.js]: ++

File: src/client/components/Sidebar/ObjectExpertiseByType/ObjectExpertiseByType.js [hunk 7]
```diff
@@ -18,20 +17,41 @@ const initialState = {
   hasNext: true,
 };
 
+const initialModalState = {
+  experts: [],
+  loading: true,
+  skip: 0,
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertiseByType/ObjectExpertiseByType.js]: use experts.length

File: src/client/components/Sidebar/ObjectExpertiseByType/ObjectExpertiseByType.js [hunk 8]
```diff
@@ -51,6 +71,19 @@ const ObjectExpertiseByType = ({ match }) => {
...
+      .then(data => {
+        setModalObjectsState({
+          ...objectsModalState,
+          experts: [...data],
+          skip: objectsModalState.skip + objectsModalState.limit,
+          loading: false,
+          hasNext: data.length === 5,
+        });
+      })
+      .catch(() => setModalObjectsState({ ...initialModalState, loading: false }));
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertiseByType/ObjectExpertiseByType.js]: remove - same request

File: src/client/components/Sidebar/ObjectExpertise.less (hunk 5 above)
Comment (konstantin1390) [reply src/client/components/Sidebar/ObjectExpertise.less]: ++

File: src/client/components/Sidebar/ObjectExpertise.js [hunk 9]
```diff
@@ -24,7 +35,7 @@ const ObjectExpertise = ({ username, wobject }) => {
       </h4>
       <div className="SidebarContentBlock__content">
         {users &&
-          _.map(_.slice(users, 0, 5), u => (
+          _.map(_.slice(users, 0, users.length), u => (
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertise.js]: No sense

File: src/client/components/Sidebar/ObjectExpertise.js [hunk 10]
```diff
@@ -45,14 +56,22 @@ const ObjectExpertise = ({ username, wobject }) => {
...
-            <h4 className="ObjectExpertise__more">
-              <Link to={`/object/${wobject.author_permlink}/expertise`}>
-                <FormattedMessage id="show_more_authors" defaultMessage="Show more authors" />
-              </Link>
-            </h4>
-          </React.Fragment>
+        {_.size(users) >= USERS_COUNT && (
+          <div className="ObjectExpertise__buttons-wrap">
+            <Link
+              className="ObjectExpertise__buttons-wrap-show-all-btn"
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertise.js]: ObjectExpertise__buttons-wrap-show-all-btn => ObjectExpertise__show-all-btn

File: src/client/components/Sidebar/ObjectExpertise.js [hunk 11]
```diff
@@ -45,14 +56,22 @@ const ObjectExpertise = ({ username, wobject }) => {
...
+              to={`/object/${wobject.author_permlink}/expertise`}
+            >
+              <FormattedMessage id="show_all" defaultMessage="Show all" />
+            </Link>
+            <div
+              className="ObjectExpertise__buttons-wrap-show-more-btn"
```
Comment (milklim) [inline src/client/components/Sidebar/ObjectExpertise.js]: ObjectExpertise__buttons-wrap-show-more-btn => ObjectExpertise__show-more-btn

**Comment by milklim** [comment]
need to discuss UI

**Pull request closed without merging**